### PR TITLE
RSR-1586: Setup Redirect Repo build tasks

### PR DESCRIPTION
**Pull request by styree**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,81 +44,126 @@
...
-                    Helpdesk Team at: helpdesk@commercialtribe.com</p>
-
-                <p>Thank you!</p>
-            </div>
+<body ng-controller="AppCtrl" layout="column">
+    
+    <div layout="column" flex layout-align="center center">
+        
+        <div layout="row">
+            <img src="https://d2531ae1m2uaia.cloudfront.net/images/ct-logo.png" alt="CommercialTribe Logo" class="ct-logo"/>
```
Comment (styree) [inline index.html]: Added the CT logo here to make it look like the other pages with this layout.

File: index.html [hunk 2]
```diff
@@ -42,81 +44,126 @@
...
+                        <!-- WINDOWS WEBTOP-->
+                        <img ng-if="isWin()" src="http://d2531ae1m2uaia.cloudfront.net/images/windows-icon.png" alt="windows logo" class="ct-bottom-20"/>
+                        <!-- MAC WEBTOP-->
+                        <img ng-if="isMac()" src="http://d2531ae1m2uaia.cloudfront.net/images/mac-icon.png" alt="apple logo" class="ct-bottom-20"/>
+                        <md-button ng-href="{{urls.webtop}}" class="md-raised ct-success ct-top-20">Desktop app</md-button>
+                    </div>
+
+                    <!-- WEB-->
+                    <div ng-if="canRunWeb()" flex="50" layout="column" layout-align="start-center">
+                        <svg width="136px" height="136px" viewBox="0 0 136 136" version="1.1" xmlns="http://www.w3.org/2000/svg" class="ct-bottom-20">
```
Comment (styree) [inline index.html]: Didn't want to mess with cloudfront, also, SVG is always better than images of icons. :)

File: index.html [hunk 3]
```diff
@@ -42,81 +44,126 @@
...
+<body ng-controller="AppCtrl" layout="column">
+    
+    <div layout="column" flex layout-align="center center">
+        
+        <div layout="row">
+            <img src="https://d2531ae1m2uaia.cloudfront.net/images/ct-logo.png" alt="CommercialTribe Logo" class="ct-logo"/>
         </div>
-        <!-- ngIf: skipWarning -->
-        <div layout="row" class=" layout-align-end ng-scope" ng-if="skipWarning"><a href="" ng-click="loginAway()">
-            <small>Login Away</small>
```
Comment (styree) [inline index.html]: I'm 90% sure this was supposed to say `Login Anyway`, so I changed it.

File: index.html (hunk 2 above)
Comment (styree) [inline index.html]: Also, symmetry is nice.

File: index.html (hunk 3 above)
Comment (p5150j) [inline index.html]: lol

**Pull request merged**